services/storage-users/pkg/command/uploads.go

```diff
@@ -39,7 +40,7 @@ func Uploads(cfg *config.Config) *cli.Command {
 func ListUploads(cfg *config.Config) *cli.Command {
 	return &cli.Command{
 		Name:     "list",
-		Usage:    fmt.Sprintf("Print a list of all incomplete uploads"),
+		Usage:    "Print a list of all incomplete uploads",
```

Comment:
```suggestion
		Usage:    "Print a list of all expired or aborted uploads",
```
Imho `incomplete` is not clear as you can interpret it as not complete but ongoing. Just my 2c